base/applications/screensavers/logon/lang/en-US.rc

```diff
@@ -2,6 +2,6 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 
 STRINGTABLE
 BEGIN
-    IDS_DESCRIPTION "Logon ScreenSaver"
+    IDS_DESCRIPTION "ReactOS Logo"
```

Comment:
Please revert.

Reply:
There are places where we deviate from Windows design, and this is one of those places. Please respect the description invented by someone who initially created this module. There is no practical reason for change (don't be a Serge).

Reply:
I gotta agree with the other two here.
We shouldn't change the preferred naming of a module someone else wrote, We'd like to respect previous authors work. Especially when it makes no functional difference :).

Reply:
Should we name it "Logon" then? Or at least "Logon Screen Saver"?

Reply:
Well a few dictionaries say Screensaver is valid (like screenshot). So if you HAVE to rename it I'd change it to Logon Screensaver. But I don't think it really needs to be touched. it's not going to make a difference.

Reply:
Reverted, but I added a space between "Screen" and "Saver" because it should be two words.